### DISCUSSION TITLE
Checking the ogg is opus/vorbis and use respective decoder and convert to mp3

### DISCUSSION THREAD

**Discussion by Alenroyfeild** [sbooth/SFBAudioEngine#330] [General]
- need to perform checking the ogg is opus/vorbis and use respective decoder and convert to mp3

**Comment by sbooth** [comment]
As of https://github.com/sbooth/SFBAudioEngine/commit/654a20ea9ee937d1a796f551cec19137ea70a272 content type detection is attempted.
